pkg/cmd/pr/merge/merge.go

```diff
@@ -40,6 +40,7 @@ type MergeOptions struct {
 	BodySet bool
 	Editor  editor
 
+	Admin                   bool
```

Comment:
Could this be called `UseAdmin` so it reads better? The cli flag can stay the same for simplicity 👍

Reply:
renamed